src/instagram.erl

```diff
@@ -0,0 +1,314 @@
+-module(instagram).
```

Comment:
Thanks for the catch. Good point on the naming being coupled to the implementation. Personally I prefer with `_client` vs without even though it's more verbose.